src/main/webapp/script.js

```diff
@@ -60,7 +60,9 @@ async function getChart() {
 
     // The json keys (ranges of scores) are sorted through their starting values
     Object.keys(aggregateValues).forEach(function(key) {
-      CommentSentimentTable.addRow([key.getInclusiveStart(), toRangeStringRepresentation(key), aggregateValues[key]]);  
+      var inclusiveStart = getRangeInclusiveStart(key);  
+      var exclusiveEnd = getRangeExclusiveEnd(key);
+      CommentSentimentTable.addRow([inclusiveStart, toRangeStringRepresentation(inclusiveStart, exclusiveEnd), aggregateValues[key]]);  
     });
```

Comment:
Since we don't have the notion of a `Range` object here anymore, maybe this argument should be switched to:

`inclusiveStart + ' to ' + exlusiveEnd`

If we want to keep this method to isolate this string representation logic, we should change that parameters it takes to reflect that we're no longer passing a Range object.